### ISSUE TITLE
Viewtype Watchlist overview (Watching, planned, dropped,...)

### ISSUE THREAD

**Issue by DaisyF8** [SwagOtaku/plugin.video.kaito#3]
Hi,

I love your addon. Thx so much.

I'm using Arctic Horizon skin (AH).

Most of my content in addons show as text Lists as default (no screenart needed for me). Then I use AH's custom viewtypes to set Poster View for the content TvShows and Seasons. That works for me for Watchlists Kitsu and MyAnimeList, I see poster screenart.

But the first step down from for example Kitsu is the list with watching, planned, ... This also gets displayed as Poster. But these items have in this viewtype no visible name associated with them.

Can you change the Type or whatever of that first step down ? To maybe have them NOT identified as Season or TvShow ? I don't know if this is possible, or up to me or the skin... Or fill in the titles (Watching,...). 

In Settings I have tried Menu Content Type as off and both as TvShow and Addon. Doesn't make a difference.

**Comment by icpryde** [comment]
I'm also using Artic Horizon and I can't see anything, I open the addon and its just a blank screen.

**Comment by icpryde** [comment]
![Screen Shot 2020-10-15 at 9 19 07 PM](https://user-images.githubusercontent.com/29389746/96202117-5a20aa80-0f2c-11eb-8089-44d9499e85d2.png)

So here you can see this is the default view type called Poster Row but you can't see the titles to any of the sections, until you change the view type too Lanscape List.

![Screen Shot 2020-10-15 at 9 19 27 PM](https://user-images.githubusercontent.com/29389746/96202168-850afe80-0f2c-11eb-9cf9-3f8250e77800.png)

Here are the options available for view types.

![Screen Shot 2020-10-15 at 9 19 37 PM](https://user-images.githubusercontent.com/29389746/96202182-8d633980-0f2c-11eb-8785-869f9674cb16.png)

Here is what it looks like now changed to landscape list.

The issue is once you do this and click on a category like "current" for example. it will then keep the "list" view not letting you see the posters.

![Screen Shot 2020-10-15 at 9 20 14 PM](https://user-images.githubusercontent.com/29389746/96202246-b5529d00-0f2c-11eb-8cc8-cf8d0de98a60.png)

So as you can see, you can see the anime options but none of the posters as it keeps the previous viewtype.

Now when i change it to Poster Row while in this section it will end up looking like this.

![Screen Shot 2020-10-15 at 9 20 28 PM](https://user-images.githubusercontent.com/29389746/96202301-ddda9700-0f2c-11eb-8b7a-17d78711f45e.png)

This is perfect, this is what I would like to see.

So in a nutshell, we need more viewtypes as OP mentioned, hope this helps make it more clear.

**Comment by SwagOtaku** [comment]
@DaisyF8 @icpryde checkout the update for the addon (version 0.0.2), let me know if this solves the issue

**Comment by DaisyF8** [comment]
For me Yes that did it.

I have skin default List View, ans two custom views for Kaito (TvShows and Seasons) as PosterRow.

The page first step from Kitsu, Anilist and MyAnimeList (watching,  planned, completed, ...) is now displaying as a list.

Thx


(and thx for adding Premiumizer)

**Comment by DaisyF8** [comment]
But there are remaining issues : 

your page under Search (with NewSearch and search history is still showing als posters without title.

Anilist displays okay but...

Kitsu displays posters and has correct description but the anime Title isn't showing.

MyAnimeList also no title but correct description.

**Comment by icpryde** [comment]
overall, great improvement @SwagOtaku thank you again for all your hard work! it definitely does display better now that we can change the 3 sections.

I did notice...something though.....not sure how this could be fixed to be honest but let me explain anyway.

When I open Kaito this is what I see, I leave it in List View as I think this looks best for these options:

![Screen Shot 2020-10-18 at 4 27 23 PM](https://user-images.githubusercontent.com/29389746/96384351-e9ad9f80-115e-11eb-9371-7ea537013c2e.png)

Now I'll click on Anilist then leave that part in List View as well:

![Screen Shot 2020-10-18 at 4 29 19 PM](https://user-images.githubusercontent.com/29389746/96385021-1eb9f200-115f-11eb-8e2f-989c3dc64f1f.png)

Then the next section seeing the list of TV Shows I put that in Poster Row to look like this:

![Screen Shot 2020-10-18 at 4 29 29 PM](https://user-images.githubusercontent.com/29389746/96385041-398c6680-115f-11eb-837b-20b34c67ed2a.png)

Then when I click a series I see the list and the thumbnails to the episodes:

![Screen Shot 2020-10-18 at 4 31 04 PM](https://user-images.githubusercontent.com/29389746/96385055-588af880-115f-11eb-9e0c-ab58218af519.png)

This is all really great, pretty much exactly what I was hoping it would do displays the episode, see a screenshot of the episode, tells me the episode number/season etc.

So this is where a little issue occures....now lets go back to the main screen and instead of clicking Anilist lets try a different category like "Latest" and as you can see because the settings are set for the Poster Row view that view doesn't work for this listing

![Screen Shot 2020-10-18 at 4 33 10 PM](https://user-images.githubusercontent.com/29389746/96385119-ca634200-115f-11eb-8975-1b144f2f1cd1.png)

My only suggestion would be to make "Latest" and "latest Dub" be under the "episode" content type. Then it would list the episodes in this type of format.

![Screen Shot 2020-10-18 at 4 25 25 PM](https://user-images.githubusercontent.com/29389746/96385155-131afb00-1160-11eb-8d96-d903cb5c71ef.png)

Trending now works just fine as it shows the posters.
Latest This Season works just fine as well, shows posters.
Upcoming Next Season also works just fine, shows posters.
All Time Popular works just fine as well, shows posters.

**Comment by SwagOtaku** [comment]
@DaisyF8 @icpryde checkout the update for the addon (version 0.0.5), i think i made the adjustments you guys talked about. Let me know how it is

**Comment by icpryde** [comment]
@SwagOtaku I'm testing it out, not sure if its working as intended? ... a few things.

Now that you've created the "Next Up" category which is awesome but typicaly you'd want that in a "lanscape" type position to showcase episode thumbnails. I mentioned this in the other thread but thumbnails are still not working as seen here:

![Screen Shot 2020-10-25 at 6 33 46 PM](https://user-images.githubusercontent.com/29389746/97121155-81c4ff00-16f2-11eb-9922-8762929b70a5.png)

Another thing, so now that "category is listed as "lanscape" if I go back and go to "Current" it will switch to landscape as well....which you kinda don't want since that section is better to be listed as "poster" since its just listing the "shows". If I change one it changes the other to the same thing. So right now either both have to be landscape or both have to be posters.

![Screen Shot 2020-10-25 at 6 50 00 PM](https://user-images.githubusercontent.com/29389746/97121210-e54f2c80-16f2-11eb-8422-26821bc0c8b8.png)

If I click on Latest, it shows the episodes and the episode detail now which is nice but also doesn't show the thumbnail.

**Comment by DaisyF8** [comment]
I just deleted my detailed feedback :p This is a fast rewrite.

But for now :

Latest and LatestSubbed should perhaps better be displayed as lists ? Difficult to extract Showname, and why?  Focus should be on releasename anyway I think.

Everything else I remarked now works correctly :) ( @Icepryde "changing one changes the other" Aren't you using Arctic Horizon custom viewtypes? It works okay for me.)

Except for NextUp - episode sometimes doesn't get extracted correctly and only gets displayed in the title "Showname - X/Y". X sometimes doesn't show up in the seperate field in the skin "Episode X". This for al three watchlists.

**Comment by icpryde** [comment]
@DaisyF8 can you explain a bit more what you mean by custom AH viewtypes do you do this in the skin settings or under the Kaito add-on itself? Maybe that is why i am having this issue I described because I never messed with the settings in AH with viewtypes?

**Comment by DaisyF8** [comment]
@icpryde I'll present my settings in context, because changing one thing can produce a different ui. Also : Check if it's working only by looking in a Watchlist (for example Current, Planned, ...). There are too many exceptions in the other items in Kodi.

Settings kaito
- General, Interface, SetContentType OFF

In kaito main menu, open Options of Arctic Horizon
- Set view as Square list. (This should translate down.)

Kodi, Settings, Skin, Items and views, Customize viewtypes
- at the bottom, Add plugin view (I added two to catch all exceptions; after adding they appear somewhere in the lower third of the list) 
--- Kaito - Seasons (Poster Row)
--- Kaito - TvShows (Poster Row)

And I believe you don't need to do anything to get the Episodes to display as Landscape.

Again : if this works in a Watchlist then you are set. The view of some other items (for example Latest) are subject to dev interpretation. We have seen @swagotaku implement feedback but not everything can satisfy everyone.

Atm I feel kaito descended from the heavens, being too good for us mere mortals 😀

**Comment by icpryde** [comment]
ya thats true on the viewtypes side of things i think its as good as its going to get now overall I'm happy with how things are implemented here.
